includes/functions.php

```diff
@@ -76,6 +85,17 @@ function is_enabled() {
 	$settings = get_settings( 'civil_comments' );
 	$installed = ! empty( $settings['publication_slug'] )? true : false;
 	$enabled = isset( $settings['enable'] ) && (bool) $settings['enable'] ? true : false;
+
+	/**
+	* Filter for Civil's is_enabled logic.
+	*
+	* Filter gives you access to the is_enabled boolean of the plugin.
```

Comment:
Please improve the titles and descriptions of each of the other filter docs like you did for the `civil_can_replace` block.